_docs/setup/install-kubernetes.md

```diff
@@ -231,52 +200,34 @@ kubectl create -f <(istioctl kube-inject -f <your-app-spec>.yaml)
 
 ## Uninstalling
 
-You can use the [Istio Helm chart](https://github.com/kubernetes/charts/tree/master/incubator/istio#uninstalling-the-chart) to uninstall, or follow the steps below.
-
-1. Uninstall Istio core components:
+1. If you installed Istio addons, uninstall them:
 
-   * If Istio was installed without Istio auth feature:
-
-     ```bash
-     kubectl delete -f install/kubernetes/istio.yaml
-     ```
-
-   * If Istio was installed with auth feature enabled:
+   ```bash
+   kubectl delete -f install/kubernetes/addons/
+   ```
 
-     ```bash
-     kubectl delete -f install/kubernetes/istio-auth.yaml
-     ```
 
-2. Uninstall RBAC Istio roles:
+1. Uninstall Istio core components. For the {{ site.data.istio.version }} release, the uninstall will delete the RBAC permissions, the `istio-system` namespace, and hierarchically all resources under it.
+   It is safe to ignore errors for non-existent resources, since they may have been deleted hierarchically.
 
-   * If beta version was installed:
+   * If Istio was installed with mutual TLS authentication disabled:
```

Comment:
given either file will work for all install types (as far as delete is concerned, the 1 line difference in the map between the 2 doesn't matter) - maybe we could just put 1 instruction here to keep it simple ?

Reply:
Agreed, but it is safer to always kubectl delete the file that was used for kubect apply. Anything different would be confusing for the user.